src/slip.rs

```diff
@@ -0,0 +1,200 @@
+use secp256k1::{PublicKey};
+
+/// A UTXO slip containing record of owernship of funds on the network
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub struct Slip {
+    /// Contains concrete data which is not relative to state of the chain
+    body: SlipBody,
+    /// Flag determining if the `Slip` is part of the longest chain
+    lc: bool,
+    /// `SlipSpentStatus` indicating the state of the slip in the network
+    pub spent_status: SlipSpentStatus,
+}
+
+/// An object that holds concrete data not subjective to state of chain
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub struct SlipBody {
+    /// An `Sectp256K::PublicKey` determining who owns the `Slip`
+    add: PublicKey,
```

Comment:
Would like to rename this to "to" but i guess we can keep it if this is the same in saito-lite...

Reply:
I would prefer to keep it as "add" instead of "to". As a convention, I would prefer to use "to" when there's a "from" field, single fields can be given "add" or "address"

Reply:
Spoke to Stephen about this. I was confused because i didn't realize slips are in vectors called from and to at the transaction level, so 'add' here makes sense to me now.